src/Compute/Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -450,7 +450,7 @@ function Test-VirtualMachineScaleSetUpdate
         $adminUsername = 'Foo12';
         $adminPassword = $PLACEHOLDER;
 
-        $imgRef = Get-DefaultCRPImage -loc $loc;
```

Comment:
where is the test for the specific bug this edit is fixing?

Reply:
test has been added, there was no existing test for this.